mail_connector_queue/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Mail Connector Queue
+====================
+
+This module uses the Connector Queue to send any emails generated by Odoo,
+by creating a queue job for each one.
+
+Configuration
+=============
+
+The cron task `Email Queue Manager` should be disabled, as it conflicts with
+this module.
```

Comment:
s/should/must/

Can you disable it when installing this module?

Please put this in readme/CONFIGURE.rst

Reply:
> Can you disable it when installing this module?

I considered it, but the record has `noupdate=1`; of course I can always override it using SQL, but should we?

Reply:
A post_init_hook is the best solution.

Reply:
@simahawk is there a customary place to store the previous state? Or should I just restore to active regardless?

Reply:
no way AFAIK. But we assume that we restore basic odoo behavior, no?